desk/app/channels.hoon

```diff
@@ -1113,6 +1115,14 @@
     =/  =nest:c     [kind.pole host name.pole]
     =/  =plan:c     =,(pole [(slav %ud time) ?~(reply ~ `(slav %ud -.reply))])
     (watch-said ask nest plan version.pole)
+  ::  handle proxied v3/v4 said requests (when /v0 prefix is added by forwarding)
```

Comment:
where does this happen?

Reply:
The `[%v0 version=?(%v3 %v4) %said ...` pattern match below is clearly a hallucination/error, so I'm eager to say "it doesn't, reject this whole addition out of hand". We don't want to change the shape of the subscription path, only add a new supported version, so the existing pattern match (above) should be able to handle it fine.